checkly/resource_heartbeat.go

```diff
@@ -174,10 +176,14 @@ func resourceHeartbeat() *schema.Resource {
 								}
 								return warns, errs
 							},
+							Default: "hours",
+							Description: "Possible values `seconds`, `minutes`, `hours` and `days`. (Default `hours`)",
 						},
 						"grace": {
 							Type:     schema.TypeInt,
 							Required: true,
+							Default:  30,
```

Comment:
Do you know if we're actually setting this as default? Because this one is required, so it shouldn't fallback to a default if no value is specified 🤔